packages/core/src/App/Components/Routes/binary-link.jsx

```diff
@@ -11,13 +10,11 @@ import { observer, useStore } from '@deriv/stores';
 // when binary link is imported into components present in routes config
 // or into their descendants
 const BinaryLink = observer(({ active_class, to, children, href, ...props }) => {
-    const platform_context = React.useContext(PlatformContext);
     const { common } = useStore();
     const { has_error, error } = common;
     const setError = error.setError;
-    const is_appstore = platform_context?.is_appstore;
     const path = normalizePath(to);
-    const route = findRouteByPath(path, getRoutesConfig({ is_appstore }));
+    const route = findRouteByPath(path, getRoutesConfig({ is_appstore: false }));
```

Comment:
do we need is_appstore config here?

Reply:
Thanks for highlighting this. 
I have removed this.